docs/GM-Commands.md

```diff
@@ -6,75 +6,104 @@ Note: Some commands are working only by selecting a player or a creature. These
 
 |Command|Security|Description|
 |-------|--------|-----------|
+|account|0|Syntax: .account Display the access level of your account.|
+|account addon|1|Syntax: .account addon #addon Set expansion addon level allowed. Addon values: 0 - normal, 1 - tbc, 2 - wotlk.|
 |account create|4|Syntax: .account create $account $password Create account and set password to it.|
 |account delete|4|Syntax: .account delete $account Delete account with all characters.|
-|account lock|0|Syntax: .account lock [on/off] Allow login from account only from current used IP or remove this requirement.|
 |account lock country|0|Syntax: .account lock country [on/off]. Allow login from account only from current used Country or remove this requirement.|
+|account lock|0|Syntax: .account lock [on/off] Allow login from account only from current used IP or remove this requirement.|
```

Comment:
This change makes No sense, to put the first part of the command after the full part

Reply:
Oops, looks like I missed this command. I did that with the rest of them

Reply:
Thank you